BigFootVentures/BigFootVentures.Business.DataAccess/Mapping/Management/OfficeMapper.cs

```diff
@@ -0,0 +1,94 @@
+﻿using BigFootVentures.Business.Objects.Management;
+using MySql.Data.MySqlClient;
+using System;
+using System.Collections.Generic;
+using System.Data;
+
+namespace BigFootVentures.Business.DataAccess.Mapping.Management
+{
+    public sealed class OfficeMapper : IMapper
+    {
+        #region "Public Methods"
+        public ICollection<object> ParseData(MySqlDataReader dataReader)
+        {
+            var entities = new List<object>();
+
+            while (dataReader.Read())
+            {
+                var entity = new Office
+                {
+                    ID = (int)dataReader["ID"],
+                    OwnerName = dataReader["Owner Name"] as string,
+
+                    OfficeName = dataReader["OfficeName"] as string,
+                    ExternalID = dataReader["ExternalID"] as string,
+                    GeographyType = dataReader["GeographyType"] as string,
+                    State = dataReader["State"] as string,
+                    StateAbbreviation = dataReader["StateAbbreviation"] as string,
+                    OfficeProperty = dataReader["OfficeProperty"] as string,
+                    OfficeUrl = dataReader["OfficeUrl"] as string,
+                    OfficeValueCategory = dataReader["OfficeValueCategory"] as string,
+                    Online = (Convert.ToSByte(dataReader["SendEmail"]) == 1),
+                    OnlineComments = dataReader["OnlineComments"] as string,
+                    CountryManager = dataReader["CountryManager"] as string,
+                    PublicationDateComments = dataReader["PublicationDateComments"] as string,
+                    RegistrationDateComments = dataReader["RegistrationDateComments"] as string,
+                    OppositionPeriodMonths = dataReader["OppositionPeriondMonths"] as string,
+                    SearchUrl = dataReader["SearchUrl"] as string,
+                    OfficeStatusesSource = dataReader["OfficeStatusesSource"] as string,
+                    OfficeNameArchieve = dataReader["OfficeNameArchieve"] as string,
+                    NationalNumberAssigned = (Convert.ToSByte(dataReader["NationalNumberAssigned"]) == 1),
+                    Points = dataReader["Points"] as string,
+                    PCT = (Convert.ToSByte(dataReader["PCT"]) == 1),
```

Comment:
please change all of bool from **(Convert.ToSByte(dataReader["PCT"]) == 1)** to **(dataReader["PCT"] as sbyte? ?? 0) == 1**